dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1517,6 +1517,97 @@ sources:
                   config: *unique-conditions
                   meta:
                     description: yrblt should be between 1850 and now
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_205_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '205'
+                  meta:
+                    description: 205s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_206_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '206'
+                  meta:
+                    description: 206s should be > 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_207_less_than_62_years_old
+                  min_value: cast(year(current_date) as decimal) - 62
+                  max_value: cast(year(current_date) as decimal)
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '207'
+                  meta:
+                    description: 207s should be <= 62 years old
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_208_less_than_62_years_old
+                  min_value: cast(year(current_date) as decimal) - 62
+                  max_value: cast(year(current_date) as decimal)
+                  additional_select_columns: *select-columns
+                  config: 
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
```

Comment:
Is there a way to do *select_conditions AND class = '208.
I took a look through the examples you mentioned, and it was always written out completely, but just wanted to ping this question.

Reply:
Good question! Sadly there is not a way to extend YAML aliases like that, so we have to fall back to duplicating all of the logic in cases where we want to extend it.